azure/v14.0.0/release.yaml

```diff
@@ -0,0 +1,61 @@
+# Generated with:
+# devctl release create --name 14.0.0 --base v13.0.2 --releases . --provider azure --overwrite true --component kubernetes@1.19.4 --component app-operator@3.0.0 --component chart-operator@2.6.0 --component cluster-operator@0.23.20 --component azure-operator@update-k8scc-v10
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v14.0.0
+spec:
+  apps:
+    - name: cert-exporter
+      version: 1.4.0
+    - name: chart-operator
+      version: 2.5.2
+    - componentVersion: 1.6.5
+      name: coredns
+      version: 1.2.0
+    - componentVersion: 0.7.4
+      name: external-dns
+      version: 1.5.0
+    - componentVersion: 1.9.7
+      name: kube-state-metrics
+      version: 1.3.0
+    - name: metrics-server
+      version: 1.1.1
+    - name: net-exporter
+      version: 1.9.2
+    - componentVersion: 1.0.1
+      name: node-exporter
+      version: 1.7.0
+  components:
+    - name: app-operator
+      releaseOperatorDeploy: true
+      version: 3.0.0
+    - name: azure-operator
+      catalog: control-plane-test-catalog
+      releaseOperatorDeploy: true
+      version: 5.2.1-dev
+      reference: 5.2.0-a467bce976cd398d192445024f59770a3f5a6d3e
+    - name: cert-operator
+      reference: 0.1.0-2
+      releaseOperatorDeploy: true
+      version: 0.1.0
+    - name: cluster-operator
+      releaseOperatorDeploy: true
+      version: 0.23.20
+    - name: kubernetes
+      version: 1.19.4
+    - name: containerlinux
+      version: 2605.9.0
+    - name: coredns
+      version: 1.6.5
+    - name: calico
```

Comment:
What about etcd and calico? should be bump them as well?

current latest releases:
- etcd 3.4.14
- calico 3.16.6

Reply:
Will need to do some extra legwork for calico first:
```
{"caller":"github.com/giantswarm/operatorkit/v4/pkg/resource/wrapper/retryresource/crud_resource.go:89","controller":"azure-operator-azureconfig-controller","event":"update","function":"GetDesiredState","level":"error","loop":"11","message":"retrying due to error","object":"/apis/provider.giantswarm.io/v1alpha1/namespaces/default/azureconfigs/2t7ut","resource":"blobobject","stack":{"annotation":"template.CloudConfigConfig.validation error: component `Calico` requires version following constraint `>= 3.10.0 < 3.16.0`, got `v3.16.6`","kind":"invalidConfigError","stack":[{"file":"/go/pkg/mod/github.com/giantswarm/k8scloudconfig/v10@v10.0.0/pkg/template/cloudconfig.go","line":35},{"file":"/root/project/service/controller/cloudconfig/cloud_config.go","line":159},{"file":"/root/project/service/controller/resource/blobobject/desired.go","line":161},{"file":"/go/pkg/mod/github.com/giantswarm/operatorkit/v4@v4.2.0/pkg/resource/wrapper/retryresource/crud_resource.go","line":82}]},"time":"2021-01-18T14:29:22.785689+00:00","version":"253373916"}
```